public/stylesheets/site/2.0/15-group-comments.css

```diff
@@ -86,6 +86,10 @@ fieldset.comments, .comment .userstuff {
   margin: 1.286em 0.643em 0.25em;
 }
 
+.comment .message {
+  padding: 1em;
```

Comment:
The margin should match the one requested in [AO3-6043](https://otwarchive.atlassian.net/browse/AO3-6034), which is `0.643em`.